docs/en/helm/component.md

```diff
@@ -68,8 +68,17 @@ spec:
           tag: "5.1.2"
 ```
 Helm module workload will use data in `settings` as [Helm chart values](https://github.com/captainroy-hy/podinfo/blob/master/charts/podinfo/values.yaml).
-Currently, you can learn the schema of settings by reading the `README.md` of the Helm chart, and the schema are totally align with [`values.yaml`](https://github.com/captainroy-hy/podinfo/blob/master/charts/podinfo/values.yaml) of the chart.
-We will integerate the values and generate it's [openapi-v3-json-schema](https://kubevela.io/#/en/platform-engineers/openapi-v3-json-schema.md) soon.
+You can learn the schema of settings by reading the `README.md` of the Helm
+chart, and the schema are totally align with
+[`values.yaml`](https://github.com/captainroy-hy/podinfo/blob/master/charts/podinfo/values.yaml)
+of the chart.  
+
+Additionally, Vela will fetch the `values.schema.json` file from the Chart
```

Comment:
In which case will the `values.schema.json` exist in a helm chart? Is there any standard ?

Reply:
I will add [Helm official doc](https://helm.sh/docs/topics/charts/#schema-files) here.